src/auth/strategies/two-factor-auth.strategy.ts

```diff
@@ -0,0 +1,21 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable } from '@nestjs/common';
+import { jwtConstants } from '../constants';
+import { TwoFactorAuthPayload } from './payloads/two-factor-auth.payload';
+
+@Injectable()
+export class TwoFactorAuthStrategy extends PassportStrategy(
+  Strategy,
+  '2FA.bearer',
```

Comment:
I see this string for a second time - move it to some more globally scoped constant